src/controllers/AuthenticationController.js

```diff
@@ -20,10 +18,6 @@ const jwtSecretkey =
   "4715aed3c946f7b0a38e6b534a8583628d84e96d10fbc04700770d572af3dce43625dd";
```

Comment:
this key is not critical security wise but would also be good to put in an extra file, which is not pushed to github and then read it here - maybe do a .yaml or .ini file then we can collect all tokens which come up during development here and then just have to share the file via a private channel if someone wants to start the app locally